ci/tasks/deploy_gcp.yaml

```diff
@@ -0,0 +1,27 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+    tag: alpine
+
+params:
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  BUCKET_NAME:
+
+inputs:
+- name: ons-design-system-release-artifact
+
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      export GOOGLE_APPLICATION_CREDENTIALS=~/gcloud-service-key.json
+      cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+      
+      gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+
+      gsutil -m cp -r ons-design-system-release-artifact/** gs://${BUCKET_NAME}/design-system/
```

Comment:
By default, the files are [cached for 1hr](https://cloud.google.com/storage/docs/gsutil/addlhelp/WorkingWithObjectMetadata#cache-control), is it worth caching for longer even for dev, given we will use these env for stress test etc. Impact should be negligible but for prod, the cache will be much higher than 1 hr.

Reply:
Good spot. This was copied from the original s3 sync and I didn't think to check. Given that this is a deployment of a release, it makes sense to set a long cache time. I have set it to 1 year.